apps/src/dance/DanceVisualizationColumn.jsx

```diff
@@ -107,6 +107,10 @@ class DanceVisualizationColumn extends React.Component {
       position: 'relative',
       overflow: 'hidden',
     };
+
+    const signedInOver13 = this.props.userType === 'teacher' || this.props.userType === 'student';
```

Comment:
Is this safer than `this.props.userType != 'student_y'`? If you leave as is, might be worth a comment that there is a third userType for students under 13 that we're implicitly using here.

Reply:
Good call on the comment - I'll add. I originally had '!= student_y', but based on this comment on my earlier PR (https://github.com/code-dot-org/code-dot-org/pull/25724#discussion_r230256035), I went with specifying 'teacher' and 'student' to make sure we err on the safe side of having the filter off except for the cases we specify.